test_utils/src/main.rs

```diff
@@ -178,6 +207,50 @@ fn main() -> Result<()> {
             let total_mint_shares = total_mint * usdn_divisor;
             print_int_u256_hex(total_mint_shares)?;
         }
+        Commands::ChainlinkPrice { feed_id, timestamp } => {
+            let chainlink_low_latency_api_key = std::env::var("CHAINLINK_DATA_STREAM_API_KEY")
+                .context("getting CHAINLINK_DATA_STREAM_API_KEY env variable")?;
+
+            let chainlink_user_secret = std::env::var("CHAINLINK_DATA_STREAM_API_SECRET")
+                .context("getting CHAINLINK_DATA_STREAM_API_SECRET env variable")?;
+
+            let chainlink_rest_url = std::env::var("CHAINLINK_DATA_STREAM_API_URL")
+                .context("getting CHAINLINK_DATA_STREAM_API_URL env variable")?;
+
+            const API_V1_REPORTS: &str = "/api/v1/reports";
+
+            let method = "GET";
+            let path = &format!("{API_V1_REPORTS}?feedID={feed_id}&timestamp={timestamp}");
```

Comment:
```suggestion
            let path = format!("{API_V1_REPORTS}?feedID={feed_id}&timestamp={timestamp}");
```

Normally, we don't add the reference symbol when defining values, but when we use them (on line 234). This is because otherwise you lose access to the underlying `String` in case you ever need to pass its ownership to another function for example.